dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/SubsystemFeatureDefinitionImpl.java

```diff
@@ -484,6 +484,33 @@ public String toString() {
         return icons;
     }
 
+    /*
+     * (non-Javadoc)
+     *
+     * @see com.ibm.ws.kernel.feature.provisioning.ProvisioningFeatureDefinition#getEndpointIcons()
+     */
+    @Override
+    public Collection<String> getEndpointIcons() {
+        Collection<String> icons = new ArrayList<String>();
+        String endpointIconsHeader = getHeader("Subsystem-Endpoint-Icons");
+        if (endpointIconsHeader != null) {
+            String[] endpointIcons = endpointIconsHeader.split(",");
+            String endpointIconCategory = "";
+            for (String endpointIcon : endpointIcons) {
+                String[] endpointIconDirectives = endpointIcon.split(";");
```

Comment:
These are really attributes if we are sticking to OSGi manifest header terms.  Directives are specified with `:=` where attributes just use the single char `=` for assignment.

Reply:
I actually changed the variable name from xxxAttrs to xxxDirectives for consistency with the code that processes the Subsystem-Icon header.  Easy change to make in both places.

Reply:
Changes completed.